conf/test.config

```diff
@@ -10,20 +10,24 @@
 ----------------------------------------------------------------------------------------
 */
 
+process {
+    resourceLimits = [
+        cpus: 2,
+        memory: '2.GB',
+        time: '1.h'
+    ]
+}
+
 params {
     config_profile_name        = 'Test profile'
     config_profile_description = 'Minimal test dataset to check pipeline function'
+}
 
-    // Limit resources so that this can run on GitHub Actions
-    max_cpus   = 2
-    max_memory = '6.GB'
-    max_time   = '6.h'
-
-    // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
-
-    // Genome references
-    genome = 'R64-1-1'
+singularity {
+    enabled = true
+    autoMounts = true
+    cacheDir = "/hps/nobackup/rdf/metagenomics/service-team/singularity-cache/"
```

Comment:
Codon path?

Reply:
oh yeah that was temporary because I already had all the containers on codon, didn't want to run it locally and have it take forever... I will remove it now, fully agree that a hackathon type effort to unify how we do unit tests (with small dbs) in our pipelines would be great